docs/intro/install.txt

```diff
@@ -19,13 +19,12 @@ database called SQLite_ so you won't need to set up a database just yet.
 Get the latest version of Python at https://www.python.org/downloads/ or with
 your operating system's package manager.
 
-You can verify that Python is installed by typing ``python`` from your shell;
+You can verify that Python is installed by typing ``python3`` from your shell;
```

Comment:
Is this still a valid concern? Do people really start Python 2 by accident?

Reply:
> Is this still a valid concern? Do people really start Python 2 by accident?

No but if you write `python` in the terminal in Linux or macOS you get an error.

Reply:
I wasn't expecting that, thanks for the info.

Reply:
`python3` fails on windows, and sometimes `python` fails as well in favor of `py` 🤔

Reply:
So, at least, we agree that `python` fails on Linux/macOS/Windows, and it's beeter to replace it.

We can replace that part with a console block that already handle variants for Linux/macOS/Windows.